src/vivarium_csu_alzheimers/data/loader.py

```diff
@@ -52,6 +52,7 @@ def get_data(
         data_keys.POPULATION.DEMOGRAPHY: load_demographic_dimensions,
         data_keys.POPULATION.TMRLE: load_theoretical_minimum_risk_life_expectancy,
         data_keys.POPULATION.ACMR: load_standard_data,
+        data_keys.POPULATION.LIVE_BIRTH_RATE: load_standard_data,
```

Comment:
where is this being used? is it needed for CrudeBirthRate?

Reply:
Yes